com.unity.formats.fbx/Editor/ExportSettings.cs

```diff
@@ -8,42 +8,88 @@
 using System.Runtime.Serialization;
 using UnityEditor.Presets;
 
-namespace UnityEditor.Formats.Fbx.Exporter {
-
+namespace UnityEditor.Formats.Fbx.Exporter
+{
     /// <summary>
     /// FBX export format options.
     /// </summary>
-    public enum ExportFormat { ASCII = 0, Binary = 1 }
+    public enum ExportFormat
+    {
+        /// <summary>
+        /// Output the FBX in ASCII format.
+        /// </summary>
+        ASCII = 0,
+        /// <summary>
+        /// Output the FBX in Binary format.
+        /// </summary>
+        Binary = 1
+    }
 
     /// <summary>
     /// Options for the type of data to include in the export
     /// (Model only, animation only, or model and animation).
     /// </summary>
-    public enum Include { Model = 1, Anim = 2, ModelAndAnim = 3 }
+    public enum Include
+    {
+        /// <summary>
+        /// Export the model without animation.
+        /// </summary>
+        Model = 1,
+        /// <summary>
+        /// Export animation only.
+        /// </summary>
+        Anim = 2,
+        /// <summary>
+        /// Export both the model and animation.
+        /// </summary>
+        ModelAndAnim = 3
+    }
 
     /// <summary>
     /// Options for the position to use for the root GameObject.
-    /// 
-    /// Local Centered: For a single root, uses the local transform information.
-    /// If you select multiple GameObjects for export, the FBX Exporter centers GameObjects 
-    /// around a shared root while keeping their relative placement unchanged.
-    /// 
-    /// World Absolute: Uses the world position of the GameObjects.
-    /// 
-    /// Reset: Exports the object to (0,0,0).
     /// </summary>
-    public enum ObjectPosition { LocalCentered = 0, WorldAbsolute = 1, Reset = 2 /* For convert to model only, no UI option*/}
+    public enum ObjectPosition
+    {
+        /// <summary>
+        /// For a single root, uses the local transform information.
+        /// If you select multiple GameObjects for export, the FBX Exporter centers GameObjects
+        /// around a shared root while keeping their relative placement unchanged.
+        /// </summary>
+        LocalCentered = 0,
+        /// <summary>
+        /// Uses the world position of the GameObjects.
+        /// </summary>
+        WorldAbsolute = 1,
+        /// <summary>
+        /// Exports the object to (0,0,0).
+        /// </summary>
+        Reset = 2 /* For convert to model only, no UI option*/
```

Comment:
I think this comment should be in the Reset summary doc.

Reply:
done